### ISSUE TITLE
inconsistent_property_type is too aggressive, with no individual way to skip

### ISSUE THREAD

**Issue by hudlow** [IBM/openapi-validator#181]
Especially where generic, context-sensitive property names are used, such as `code`, `inconsistent_property_type` property type can lead to false positives across unrelated schemas. I believe we ought to rethink how this check works, or provide a way to skip the check for specific properties.

**Comment by dpopp07** [comment]
The proposed solution here is to provide a list of values that we ignore this warning for. This could also be made configurable.

**Comment by jorge-ibm** [comment]
Mirrored issue on sdk board: https://github.ibm.com/arf/planning-sdk-squad/issues/2375

**Comment by hudlow** [comment]
Not a very long list, but here are the property names I am familiar with that run afoul of this rule despite somewhat legitimate reasons for having varying types:
```
default
value
code
type
```
